mne/export/_edf.py

```diff
@@ -206,6 +206,12 @@ def _export_raw(fname, raw, physical_range, add_ch_type):
     # compute number of data records to loop over
     n_blocks = n_times / out_sfreq
 
+    # increase the number of annotation signals if necessary
+    n_annotations = len(raw.annotations)
```

Comment:
this will fail if `raw.annotations` is `None`.

Reply:
please don't mark conversations as "resolved" without at least commenting on why you think it's not a problem @adam2392 (or changing something if the comment is valid).  Is this indeed not a problem?

Reply:
Ah sorry, I thought I resolved it by checking if `n_anotations is None`. I realize that's not solving the issue. Let me push a fix.

Reply:
Pushed a fix.